Tensile/Source/CMakeLists.txt

```diff
@@ -51,11 +51,19 @@ if(TENSILE_NEW_CLIENT)
 
     if(TENSILE_USE_OPENMP)
         #set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_OPENMP")
-        find_package(OpenMP)
+        find_package(OpenMP QUIET)
         if (OPENMP_FOUND)
             set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
             set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
             set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
+        else ()
+            if(EXISTS /etc/redhat-release)
+                set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp=libgomp")
```

Comment:
This is incomplete. I can add CentOS/SLES support to CI at a later time. CI is Ubuntu only, and the changes in this PR will allow CI and (Tensile in general) to run with the new client